src/library_noderawfs.js

```diff
@@ -96,7 +96,7 @@ mergeInto(LibraryManager.library, {
       var newMode = NODEFS.getMode(pathTruncated);
       var fd = suggestFD != null ? suggestFD : FS.nextfd(nfd);
       var node = { id: st.ino, mode: newMode, node_ops: NODERAWFS, path: path }
-      var stream = FS.createStream({ fd: fd, nfd: nfd, position: 0, path: path, flags: flags, node: node, seekable: true });
+      var stream = FS.createStream({ nfd: nfd, position: 0, path: path, flags: flags, node: node, seekable: true }, fd, fd + 1);
```

Comment:
I don't know this code well, but it looks like the last parameter is not one past the limit, but included in the limits checked (in `nextfd()` which is what ends up called AFAICT). So should this end in `, fd, fd);` perhaps if the goal is to tell it to use `fd` exactly?

Reply:
Apparently so:
```js
for (var fd = fd_start; fd <= fd_end; fd++) {
```